package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "purescript-lens",
-  "version": "1.0.1",
```

Comment:
I interpreted this to that it's required on npm level. However isn't omitting the version  the standard for purescript (e.g. the libs in purescript organisation)? Not sure why exactly but I guess `pulp` takes care of this?  See e.g. https://github.com/purescript/purescript-lists/blob/master/package.json